docs/source/sysadmin_guide/grafana_configuration.rst

```diff
@@ -0,0 +1,150 @@
+********************************
+Configuring Grafana and InfluxDB
+********************************
+
+This procedure describes how to display Sawtooth network statistics and operating system metrics in Grafana, using Docker, InfluxDB, and Telegraf.
+
+Requirements
+============
+
+This guide assumes Docker and one or more sawtooth validators have been configured.
+
+Download sawtooth-core from the Hyperledger GitHub repository.
+
+.. code-block:: console
+
+   $ git clone https://github.com/hyperledger/sawtooth-core.git
+
+InfluxDB
+========
+
+#. First, pull the InfluxDB Docker container from Docker Hub.
+
+   .. code-block:: console
+
+     $ docker pull influxdb
+
+#. Next, start the InfluxDB Docker container with persistent storage on the local file system and port 8086 exposed to the network. We also configure the database, admin, and regular users here.
```

Comment:
"start an InfluxDB Docker container" you can create multiple containers from one image. also this line should be wrapped at 80 characters.